Engine/Alphas/DefaultAlphaHandler.cs

```diff
@@ -251,9 +252,10 @@ protected virtual void StoreInsights()
                     var insights = InsightManager.AllInsights.OrderBy(insight => insight.GeneratedTimeUtc).ToList();
                     if (insights.Count > 0)
                     {
-                        var directory = Path.Combine(Directory.GetCurrentDirectory(), AlgorithmId);
-                        var path = Path.Combine(directory, "alpha-results.json");
+                        var baseDirectory = Config.Get("insights-destination-folder", Directory.GetCurrentDirectory());
```

Comment:
Minor but would be good to store `path` to local variable to avoid checking config 
`Config.Get("insights-destination-folder", Directory.GetCurrentDirectory());`

Reply:
Yes, fair call. I'll change it.

Reply:
Wonder if we need a separate `insights-destination-folder` than `results-destination-folder`, could they be the same? Thinking of what is the use case for having them separate

Reply:
I couldn't think of one to be honest, but figured you could always default them to the same thing, but it'd be difficult to go the other way. Happy to combine them to one value.

Reply:
I've followed your question I've merged the insights directory location into results directory location. It can always be spun out later if required.